docs/Using-Fleet/Mobile-device-management.md

```diff
@@ -2,14 +2,58 @@
 
 MDM features are not ready for production and are currently in development. These features are disabled by default.
 
-MDM features allow you to mange macOS updates and macOS settings on your hosts.
+MDM features allow you to manage macOS updates and macOS settings on your hosts.
 
-To use MDM features you have to connect Fleet to Apple Push Certificates Portal:
+## Apple Push Notification Service (APNs)
 
-1. In the Fleet UI, head to the **Settings > Integrations > Mobile device management (MDM)** page. Users with the admin role can access the settings pages.
+To use MDM features you have to connect Fleet to Apple Push Certificates Portal. This can be done via the Fleet UI or the `fleetctl` command-line interface. The result is the same regardless of the method used, you end up with 4 files: an APNs certificate and private key files along with the Simple Certificate Enrollment Protocol (SCEP) certificate and private key files. Make sure you store them securely as you will need them to configure your Fleet instances afterwards (and take note of the email and organization used to generate them, as you will need to use the same ones during renewal). For _renewal_ of an existing but expired (or soon to expire) APNs certificate, see the [APNs Renewal section](#apns-renewal) below.
 
+Via the Fleet UI:
+1. Head to the **Settings > Integrations > Mobile device management (MDM)** page. Users with the admin role can access the settings pages.
 2. Follow the instructions under **Apple Push Certificates Portal**.
 
+Via the `fleetctl` CLI:
+1. Run `fleetctl generate mdm-apple --email <email> --org <org>`.
+2. Follow the on-screen instructions.
```

Comment:
```suggestion
2. Follow the on-screen instructions.

> Take note of the Apple ID you use to sign into Apple Push Certificates Portal. You'll need to use the same Apple ID when renewing your APNs certificate. Apple requires that APNs certificates are renewed once every year. To renew, see the [APNs Renewal section](#apns-renewal) .
```

@mna do you know if the `--email` and `--org` matter? My understanding is that the Apple ID used to create the APNs cert matters.

Reply:
I think you removed the mention to store the certs/keys securely for use with Fleet's config later on:

> Make sure you store them securely as you will need them to configure your Fleet instances afterwards

Was that on purpose?

Reply:
> do you know if the --email and --org matter?

Yeah I'm not sure about the --org, the documentation elsewhere only mentions the Apple ID, so that may be the only requirement.

Reply:
> Yeah I'm not sure about the --org, the documentation elsewhere only mentions the Apple ID, so that may be the only requirement.

Got it. @roperzh do you know? Can we test this? 

For now, let's merge this as (Apple ID is the only requirement). If we're wrong we can come back and change.

Reply:
> I think you removed the mention to store the certs/keys securely for use with Fleet's config later on

@mna this was intentional because I think it's implied. As a user following the instructions, I generate the certs/keys and then the next step (or the step after) is configuring them.

Please feel free to add it back if you disagree (you think it's not implied).

Reply:
> As a user following the instructions, I generate the certs/keys and then the next step (or the step after) is configuring them.
> 
> Please feel free to add it back if you disagree (you think it's not implied).

That's probably fine, that was just as an extra precaution, but I like how terse your proposals are.

Reply:
@noahtalerman @mna the org it's not required to generate the APNs key pair, but it is required by `fleetctl`, this was a requirement in the issue.

This was also briefly discussed here: https://github.com/fleetdm/fleet/pull/8812#discussion_r1035394426